languages/go/exercises/concept/basic-slices/.meta/design.md

```diff
@@ -0,0 +1,41 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics on how to work with slices in Go.
+
+## Things to teach
+
+For example:
+
+- instantiate (len, cap)
+- append
+- set value
+- read value
+- remove value
+- taking length
+- subslices
+- expanding slices (e.g. in `append(slice1, slice2...)`)
+
+## Things not to teach
+
+For example:
+
+- inner workings of slices
+- arrays
+
+## Concepts
+
+- `basic-slices`
+
+## Prerequisites
```

Comment:
Isn't arrays also a prerequisite here?

Reply:
We are not using arrays, no. They are technically underlying, but since you almost never use them in Go, I would handle them after and in a separate branch that nothing (?) relies on.

Reply:
Okay, clear